src/index.js

```diff
@@ -0,0 +1,59 @@
+import * as task from './modules/status.js';
+import './style.css';
+
+let list = [
+  { description: '', isCompleted: false, index: 1 },
+  // { description: 'Set up a new project with webpack', isCompleted: false, index: 1 },
+  // { description: 'Create an index.js file', isCompleted: false, index: 2 },
+  // { description: ' over the tasks array and populate an HTML', isCompleted: false, index: 3 },
+];
```

Comment:
Hi @Bludidi 

You did a great job. But there is something you can do to improve your project.
✔Good GitHub flow.
✔No Linter Error.
✔Good README.md file.

Please check the comments below.
- It will be better if you remove this commented code because will
   looks professional.